pkg/healthcheck/healthcheck.go

```diff
@@ -786,7 +786,7 @@ func (hc *HealthChecker) allCategories() []category {
 				},
 				{
 					description: "issuer cert is issued by the trust root",
-					hintAnchor:  "l5d-certs-rotation",
+					hintAnchor:  "l5d-identity-issuer-cert-issued-by-trust-root",
```

Comment:
I've noted the discrepancy against the docs [here](https://github.com/linkerd/website/pull/609/files#r359996909)

Reply:
yes... that is correct. Thanks!  Its a typo in the docs PR, not this one